### DISCUSSION TITLE
Highlighting calendar tasks in different colors depending on priority ?

### DISCUSSION THREAD

**Discussion by IgorA100** [YetiForceCompany/YetiForce#58] [How to] [answered]
Highlighting calendar tasks in different colors depending on priority

Is it possible to somehow highlight tasks with high priority in text or background color?
It would be great if it was possible to color both in the calendar itself, and in the list of entries and in the widget.
I thought, read, but still couldn't. Is it possible to do this without editing the code?

**Comment by rskrzypczak** [comment, marked as answer]
in simple words - no, you can't
calendar element colors only include: calendar type, user color

**Reply by IgorA100** [reply]
Ok.

**Comment by IgorA100** [comment]
Are there any plans to implement coloring in the calendar based on task priority?
I think that would be very useful!
